content/en/docs/collector/configuration.md

```diff
@@ -241,7 +249,7 @@ receivers:
         - job_name: otel-collector
           scrape_interval: 5s
           static_configs:
-            - targets: [localhost:8888]
+            - targets: [0.0.0.0:8888]
```

Comment:
I think this should be reverted as well since this is a scrapping info (pull, not push)